extensions/spaces/libspaces.m

```diff
@@ -131,26 +131,28 @@ static int spaces_windowsForSpace(lua_State *L) { // NOTE: wrapped in init.lua
     return 1 ;
 }
 
-/// hs.spaces.moveWindowToSpace(window, spaceID) -> true | nil, error
+/// hs.spaces.moveWindowToSpace(window, spaceID, force) -> true | nil, error
```

Comment:
because `force` is optional, the syntax here should really be:

```/// hs.spaces.moveWindowToSpace(window, spaceID[, force])```

Reply:
Fixed.